skia/RSkSVGContainer.cpp

```diff
@@ -160,19 +162,6 @@ void RSkSVGContainer::setRoot(RSkSVGNode * rootNode) {
   }
 }
 
-void RSkSVGContainer::removeDefEntry(std::string key) {
```

Comment:
This APi is no more needed, beacuse of below reason.

Def Entry of children will be Merged to parent container [SVgView] and only the Entry in SvgView will be accessed across.
Locally Maintained def entries in child element stays as stale and it is removed only when unmount happens using this API. To avoid this removing children's def entry once after merging it with parent.